ui/rla_ui.js

```diff
@@ -8,968 +8,996 @@
 //   (Either define this more carefully or just limit to "a-z0-9_-")
 
 
-(function(){
```

Comment:
@AbnerSD there's "diff noise" here making the real changes harder to find. I believe your editor is changing spaces to tabs, and using `"\r\n"` newlines instead of `"\n"`. I can add `?w=0` to the url to help with the diff a bit, but would you mind changing this so the git history is easier to read?

It's probably an editor setting (e.g. for replacing tabs), but you can also use something like this:

    sed -i 's/\r//' ui/rla_ui.js

Reply:
Ah, after further research it looks like this may be another way to solve the `"\r\n"` problem:

https://git-scm.com/docs/git-config#Documentation/git-config.txt-coreautocrlf
https://help.github.com/en/articles/dealing-with-line-endings

In other words, set `core.autocrlf` to `true`, and it will use `"\r\n"` locally but then commit with `"\n"`.